sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/package.scala

```diff
@@ -162,7 +162,18 @@ package object util {
   def toCommentSafeString(str: String): String = {
     val len = math.min(str.length, 128)
     val suffix = if (str.length > len) "..." else ""
-    str.substring(0, len).replace("*/", "\\*\\/").replace("\\u", "\\\\u") + suffix
```

Comment:
We only need to make sure that the comment string does not have `*/` in it, one simpler solution could be

```
str.substring(0, len).replace("*/", "*\\/").replace("u0022/", "u0022\\/") + suffix
```